package.json

```diff
@@ -51,22 +51,23 @@
     "peer-book": "~0.5.1",
     "pre-commit": "^1.2.2",
     "pull-goodbye": "0.0.2",
+    "pump": "^2.0.0",
```

Comment:
Why add an unused dep?

Reply:
Hmm, it was failing to run on ci because apparently https://github.com/libp2p/js-libp2p-multiplex/blob/fa78df420ca5762b2a003e658fd74bdf1dd3e6b3/src/index.js#L9 is failing to include it as a dep - https://github.com/libp2p/js-libp2p-multiplex/blob/master/package.json#L31..L48. I'll patch `js-libp2p-multiplex`.

Reply:
this should fix the pump issues, as well as port multiplex to use the new aegir https://github.com/libp2p/js-libp2p-multiplex/pull/70